packages/icons/src/icons.ts

```diff
@@ -16,6 +17,7 @@ export const icons = {
     close: require('../assets/icons/close.svg'),
     closeCircle: require('../assets/icons/closeCircle.svg'),
     eject: require('../assets/icons/eject.svg'),
+    discover: require('../assets/icons/discover.svg'),
```

Comment:
Did you use generate scripts for generating this? It seems it's not in alphabetically order

Reply:
Yeah I used the script. I'll try to delete this and generate again. If it's same, it's probably just error in the lib.

Reply:
From what I see it is not sorted in script itself because it uses same order in which files are loaded. If this at least stable and does sort in same way everywhere it should be enough.

Reply:
[fdff860](https://github.com/trezor/trezor-suite/pull/5678/commits/fdff8602ae6d0842e61a03fa3ab9e2f524ee15e5)